src/components/EventForm.js

```diff
@@ -0,0 +1,124 @@
+import React from "react";
+import "react-dates/initialize";
+import "react-dates/lib/css/_datepicker.css";
+import { DateRangePicker } from "react-dates";
+import moment from "moment";
+import uuid from "uuid";
+
+export default class EventForm extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      title: "",
+      description: "",
+      organisator: "",
+      city: "",
+      category: "",
+      image: false,
+      id: uuid(),
+      createdAt: moment(),
+      startDate: moment(),
+      startDateId: "",
+      endDate: moment(),
+      endDateId: "",
+      calendarFocused: null,
+    };
+  }
+
+  onTitleChange = e => {
+    const title = e.target.value;
+    this.setState(() => ({ title }));
+  };
+
+  onDescriptionChange = e => {
+    const description = e.target.value;
+    this.setState(() => ({ description }));
+  };
+
+  onOrganisatorChange = e => {
+    const organisator = e.target.value;
+    this.setState(() => ({ organisator }));
+  };
+
+  onCityChange = e => {
+    const city = e.target.value;
+    this.setState(() => ({ city }));
```

Comment:
If you don't use `prevState` then `this.setState({ city })` would be sufficient.